### DISCUSSION TITLE
How cloud I select the jsonb[] contains one of string in array?

### DISCUSSION THREAD

**Discussion by liaojiwei666** [prisma/prisma#16774] [Q&A] [answered]
for example, I had dataset like below:
| id | groups |
|---|------|
| 1 | [ 'A' ] |
| 2 | [ 'B' ] |
| 3 | [ 'A', 'C' ] |
| 4 | [ 'B', 'D' ] |
| 5 | [ 'C' ] |
| 6 | [ 'D' ] |
| 7 | [ 'A', 'B' ] |

And I want select groups contain one of ['A' | 'B'].
The result must be id [1, 2, 3, 4, 7]

The raw sql present like `(groups)::jsonb ?| array['A', 'B']`

I use the ORM way like below
```js
const chosen = await prisma.table.findMany({
  where: {
    groups: {
      array_contains: ['A', 'B']
    }
  }
})
```
but the result I got only the `id:7`.
this is not what I want...
How could I implement my theory by way of ORM?

**Comment by liaojiwei666** [comment, marked as answer]
Solved:
use OR to union conditions.
```js
const chosen = await prisma.table.findMany({
    where: {
        OR: [
            {
                groups: {
                    array_contains: ['A']
                }
            },
            {
                groups: {
                    array_contains: ['B']
                }
            },
        ]
    }
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
